modules/primer-marketing-utilities/docs/borders.md

```diff
@@ -17,3 +17,22 @@ Top, right, bottom, and left border utilities are can be used responsively to ad
   .border-top-0
 </div>
 ```
+
+## Border Colors
```

Comment:
Does this need to be changed to `Marketing border colors` to avoid dupes in stories?

Reply:
I have these added manually to the story for now, using the class names. But they're under "marketing utilities" so I think it's pretty clear.